src/status-indicator.scss

```diff
@@ -46,12 +46,16 @@ $color-states: (
     flex-direction: unset;
```

Comment:
should be `row` if it's default. Inherit, initial, unset have problems with IE11

Reply:
&:focus {
    outline: 1px dotted #000;
    outline-offset: 1px;
  }

you can't have 1px and #000  hardcoded

Reply:
there's a mixin for the focus that you can use

Reply:
Fixed